app/src/main/java/org/simple/clinic/bloodsugar/BloodSugarMeasurement.kt

```diff
@@ -91,6 +91,17 @@ data class BloodSugarMeasurement(
     @Query("SELECT * FROM BloodSugarMeasurements WHERE syncStatus = :status")
     fun withSyncStatus(status: SyncStatus): List<BloodSugarMeasurement>
 
+    @Query("""
+      SELECT * FROM bloodsugarmeasurements
+      WHERE syncStatus = :syncStatus
+      LIMIT :limit OFFSET :offset
+    """)
+    fun withSyncStatusBatched(
```

Comment:
The naming convention for other syncStatus batched is `dao#recordsWithSyncStatusBatched`. Can you add the same here as well. It'll be consistent throughout

Reply:
Good point. Will do this.

Reply:
Done.